cmd/notary/tuf.go

```diff
@@ -285,7 +320,7 @@ func (t *tufCommander) tufInit(cmd *cobra.Command, args []string) error {
 		cmd.Printf("Root key found, using: %s\n", rootKeyID)
 	}
 
-	if err = nRepo.Initialize(rootKeyID); err != nil {
+	if err = nRepo.Initialize([]string{rootKeyID}); err != nil {
```

Comment:
If other root keys already exist on this host, this `rootKeyID` may not necessarily correspond to the `--rootkey` we pass in.  We should change the conditional logic above such that it prioritizes keys from the `--rootkey` flag if we passed any in